exporter/awsxrayexporter/translator/cause_test.go

```diff
@@ -74,7 +75,48 @@ func TestCauseWithHttpStatusMessage(t *testing.T) {
 	assert.True(t, strings.Contains(jsonStr, errorMsg))
 }
 
-func constructExceptionServerSpan(attributes map[string]interface{}) pdata.Span {
+func TestCauseWithZeroStatusMessage(t *testing.T) {
+	errorMsg := "this is a test"
+	attributes := make(map[string]interface{})
+	attributes[semconventions.AttributeHTTPMethod] = "POST"
+	attributes[semconventions.AttributeHTTPURL] = "https://api.example.com/widgets"
+	attributes[semconventions.AttributeHTTPStatusCode] = 500
+	attributes[semconventions.AttributeHTTPStatusText] = errorMsg
+
+	span := constructExceptionServerSpan(attributes, tracetranslator.OCOK)
```

Comment:
Hmm, I'm not an expert of Go but I'm surprised we can pass an int as `StatusCode` without a cast.

Anyways, we are using `pdata.StatusCode` so we should use it, not `tracetranslator` types. They're here

https://github.com/open-telemetry/opentelemetry-collector/blob/a507cf9faf7d9a3bb6ff40b67426e72f9c78de38/internal/data/opentelemetry-proto-gen/trace/v1/trace.pb.go#L91

Alias from pdata is defined here

https://github.com/open-telemetry/opentelemetry-collector/blob/a507cf9faf7d9a3bb6ff40b67426e72f9c78de38/consumer/pdata/trace.go#L127

For some background, the collector started as a fork from a project called opencensus. The opentelemetry data model shares a lot with it but has some differences and is the new protocol we are using, things like tracetranslator are there to support code that still hasn't migrated yet. That's why the name is prefixed `OC` for opencensus. But we don't want to use opencensus code in our package since we're already using the new model.